frontend/public/components/storage/shared.ts

```diff
@@ -100,10 +100,10 @@ export const provisionerAccessModeMapping: ProvisionerAccessModeMapping = {
     Filesystem: ['ReadWriteOnce'],
     Block: ['ReadWriteOnce'],
   },
-  'openshift-storage.cephfs.csi.ceph.com': {
+  'cephfs.csi.ceph.com': {
     Filesystem: ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
   },
-  'openshift-storage.rbd.csi.ceph.com': {
```

Comment:
Lets add a property under provisioner for partial matching. `partialMatch` we do partial matching in `getProvisionerAccessModeMapping` only on such cases.

Reply:
also just to point out even console uses partial check for provisioners

Reply:
and if u mean this ??:
```
  'cephfs.csi.ceph.com': {
    Filesystem: ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
    partialMatch: true,
  },
```
then issue is, say in ROSA case:
`provisioner` will be `rosa-namespace.cephfs.csi.ceph.com`, but key in provisioners map is `cephfs.csi.ceph.com`, so we will not be able to read property `partialMatch` as it would never match.

Reply:
updated as per suggestion...